src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogBuildListener.java

```diff
@@ -154,6 +160,21 @@ public void onCompleted(Run run, @Nonnull TaskListener listener) {
             Map<String, Set<String>> tags = buildData.getTags();
             String hostname = buildData.getHostname("unknown");
             client.gauge("jenkins.job.duration", buildData.getDuration(0L) / 1000, hostname, tags);
+            logger.fine(String.format("[%s]: Duration: %s", buildData.getJobName(null), toTimeString(buildData.getDuration(0L))));
+
+            if (run instanceof WorkflowRun) {
+                RunExt extRun = getRunExtForRun((WorkflowRun) run);
+                long pauseDuration = 0;
+                for (StageNodeExt stage : extRun.getStages()) {
+                    pauseDuration += stage.getPauseDurationMillis();
+                }
+                client.gauge("jenkins.job.pauseduration", pauseDuration / 1000, hostname, tags);
```

Comment:
to match the stage duration names we should update all instances of these metrics like so:
```suggestion
                client.gauge("jenkins.job.pause_duration", pauseDuration / 1000, hostname, tags);
```

Reply:
Sure, makes sense 👍